--- gkII-0.4.7/src/Makefile	2009-12-27 05:58:32.000000000 +0000
+++ gkII-0.4.7.tpg/src/Makefile	2009-12-31 17:57:48.000000000 +0000
@@ -9,7 +9,7 @@
 
 #CFLAGS := $(DEFS) -Wall -g2 #-DDEBUG_PAR
 
-CFLAGS := $(DEFS) -O2 -fomit-frame-pointer -Winline
+CFLAGS += $(DEFS) -O2 -fomit-frame-pointer -Winline
 LIBS   := -lpng -lz
 
 SRC    := $(wildcard *.c)
@@ -18,7 +18,7 @@
 
 $(PROG): $(OBJS)
 	$(CC) `pkg-config --cflags gtk+-2.0` $(CFLAGS) $(OBJS) -o $(PROG) \
-	`pkg-config --libs gtk+-2.0` $(LIBS)
+	`pkg-config --libs gtk+-2.0` $(LDFLAGS) $(LIBS)
 	
 %.o : %.c $(HEADERS)
 	$(CC) `pkg-config --cflags gtk+-2.0` $(CFLAGS) -c $<
